ONSdigital/response-operations-ui
# Dynamically populate secure messaging drop down

# Motivation and Context
At present every time we onboard a new survey we manually have to update the dropdown box in r-ops to reflect the change so secure messaging can be used. The box should be dynamic and call from the survey service to populate it. 

# What has changed
- Secure-message dropdown now gets populated from the survey service
- FDI Grouping happens in Config instead of its own file.
- Added specific FDI create message and messages page tests

# How to test?
- Run r-ops tests and the acceptance tests
- Check the secure-message dropdown works and is populated with every survey
- Set a FDI survey collection exercise to ready to live, send a message and you should be directed to the FDI messages screen. Try with multiple FDI surveys.
- Create a survey using `/surveys/create` and it should be put into the surveys dropdown box
# Links
[Trello Card](https://trello.com/c/Xfc0UL3i/326-dynamically-populate-secure-messaging-drop-down-m)
